lib/screens/settings/components/account_card.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:hashed/domain-shared/ui_constants.dart';
+import 'package:hashed/utils/ThemeBuildContext.dart';
+import 'package:hashed/utils/short_string.dart';
+
+class AccountCard extends StatelessWidget {
+  final String address;
+  final String? accountName;
```

Comment:
can this ever be null? 
looks to me that an AccountCard should always have an account

Reply:
Yes it always has an address, but we will fetch account names, and they can be null (ie there's no name set on the identity pallet)